<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Minimalist Recorder</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>220</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Recording Control</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="recordButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>180</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Record</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>50</y>
      <width>180</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QFrame" name="line1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>500</width>
      <height>2</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::HLine</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
   </widget>
   <widget class="QLabel" name="startDateLabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Time:</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="startDateEdit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>150</y>
      <width>250</width>
      <height>25</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>dd/MM/yyyy HH:mm:ss</string>
    </property>
   </widget>
   <widget class="QLabel" name="endDateLabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>End Time:</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="endDateEdit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>190</y>
      <width>250</width>
      <height>25</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>dd/MM/yyyy HH:mm:ss</string>
    </property>
   </widget>
   <widget class="QFrame" name="line2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>500</width>
      <height>2</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::HLine</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="visualizeButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>260</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Visualize</string>
    </property>
   </widget>
   <widget class="QPushButton" name="settingsButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>300</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Settings</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>